api/data_explorer/controllers/export_url_controller.py

```diff
@@ -236,20 +236,18 @@ def _get_range_clause(column, value):
 def _get_facet_and_value(filter):
     facets = current_app.config['UI_FACETS']
     split = filter.rsplit('=', 1)
-    return facets[split[0]], split[1]
+    return split[0], facets[split[0]], split[1]
 
 
-def _get_table_and_clause(facet, value, sample_file_column_fields):
+def _get_table_and_clause(es_field_name, type, value,
+                          sample_file_column_fields):
     """Returns a table name and a single condition of a WHERE clause,
     eg "((age76 >= 20 AND age76 < 30) OR (age76 >= 30 AND age76 < 40))".
     """
     sample_file_type_field = False
-    es_field_name = ''
-    if facet['type'] == 'samples_overview':
+    if type == 'samples_overview':
```

Comment:
nit: `type` is [built-in](https://docs.python.org/2/library/functions.html#type) python function and should be avoided as a variable name.